docarray/index/backends/hnswlib.py

```diff
@@ -206,6 +217,15 @@ def python_type_to_db_type(self, python_type: Type) -> Any:
             if safe_issubclass(python_type, allowed_type):
                 return np.ndarray
 
+        type_map = {
```

Comment:
so we must be clear on what items are filterable and which are not. To be mindful when documenting

Reply:
I'll update docs here https://github.com/docarray/docarray/pull/1678